keyboards/clog/keyboard.json

```diff
@@ -0,0 +1,90 @@
+{
+    "keyboard_name": "The Clog",
+    "manufacturer": "S'mores",
+    "url": "https://github.com/smores56/clog",
+    "maintainer": "@smores56",
+    "usb": {
+      "vid": "0xBEEF",
+      "pid": "0x5051",
+      "device_version": "0.0.1"
+    },
+    "development_board": "elite_c",
+    "pin_compatible": "elite_c",
```

Comment:
Sorry for the churn on this suggestion. Looking at it again, and looking at the linked repos, this should be more correct. 
```suggestion
```